LICENSE

```diff
@@ -1,4 +1,192 @@
-   Copyright 2018-2021 Equinix, Inc.
+                                 Apache License
```

Comment:
Do we need anyone to sign off on this license change?

Reply:
Nevermind, it's the same license, your PR just adds the rest of the license text.

Reply:
Moving from Apache to Apache. The difference is the copyright range and name, which in this PR is updated with the LICENSE template text. We should add Kubernetes Authors too (unless there is a newer convention).

Here's an example: https://github.com/linode/linode-cloud-controller-manager/blob/master/LICENSE#L189-L192

Reply:
It's also pretty common, to leave this as the template text.. https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/LICENSE#L189